test/runtests.jl

```diff
@@ -13,6 +13,7 @@
 using JuMP
 using Compat
 using FactCheck
+using Base.Test
```

Comment:
``@test_approx_eq`` is used somewhere in the nonlinear tests, and was failing locally when this wasn't here. Not sure why I haven't seen it before...